src/litdata/streaming/serializers.py

```diff
@@ -387,13 +388,36 @@ def can_serialize(self, data: float) -> bool:
         return isinstance(data, float)
 
 
+class TIFFSerializer(Serializer):
+    """Serializer for TIFF files using tifffile."""
+
+    def serialize(self, item: Any) -> Tuple[bytes, Optional[str]]:
+        if not isinstance(item, str) or not os.path.isfile(item):
+            raise ValueError(f"The item to serialize must be a valid file path. Received: {item}")
+
+        # Read the TIFF file as bytes
+        with open(item, "rb") as f:
+            data = f.read()
+
+        # Store metadata if needed
+        _, ext = os.path.splitext(item)
+        metadata = f"tifffile:{ext.lower()}"
+        return data, metadata
```

Comment:
You don t need metadata. It is used onlt when restoring data conditional like dtype for tensors.

Reply:
Seems necessary to return None